src/coreclr/hosts/corerun/corerun.cpp

```diff
@@ -25,13 +26,29 @@ static const wchar_t *serverGcVar = W("COMPlus_gcServer");
 // On by default.
 static const wchar_t *concurrentGcVar = W("COMPlus_gcConcurrent");
 
+// Environment variable for setting whether to swallow uncaught exceptions (for testing)
```

Comment:
We have links to CoreRun in the docs as reference implementations of the hosts: https://docs.microsoft.com/en-us/dotnet/core/tutorials/netcore-hosting#sample-hosts

What people do with this is that will copy&paste the corerun code and then keep going with it...

Reply:
Hmmm…  I guess that means, I will switch back to environment variables for the test.